string name
string type

string aggMethod
string andMethod
string orMethod
string impMethod
string defuzzMethod

irols/Var[] input
irols/Var[] output

irols/Rule[] rules
